src/Symfony/Component/Serializer/Mapping/Loader/XmlFileLoader.php

```diff
@@ -68,7 +68,13 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata): bool
                 }
 
                 if (isset($attribute['serialized-name'])) {
-                    $attributeMetadata->setSerializedName((string) $attribute['serialized-name']);
+                    $attributeMetadata->setSerializedName((string) $attribute['serialized-name'], []);
```

Comment:
```suggestion
                    $attributeMetadata->setSerializedName((string) $attribute['serialized-name']);
```
As it's already the default one